cmp/ag-grid/AgGrid.scss

```diff
@@ -67,11 +67,11 @@
       background-color: var(--xh-grid-group-bg);
     }
 
-    .group-header-icon-expanded {
+    .ag-group-expanded {
       margin: 0 var(--xh-pad-half-px);
```

Comment:
There are two usages of 'ag-group-expanded' in this file, both of which seem to be doing the same thing (i.e. wiping out the left margin). Is this redundant?

Reply:
I don't think so - this one here is adding a left margin - we have a 0 top/bottom and a half-pad left/right margin being applied on this line.

Below we have the rule nested under ` .xh-grid--hierarchical` - see the comment on line 213.